src/region/RegionCreator.tsx

```diff
@@ -108,6 +107,11 @@ export default function RegionCreator({ className, onAbort, onStart, onStop }: P
         positionY2Ref.current = null;
         regionDirtyRef.current = true;
 
+        if (!hasStartedRef.current) {
```

Comment:
I don't think a guard clause here is appropriate since most logic has already run. Can we just call the `onStop` or `onAbort` callback and defer to the parent to figure it out?